From bef68ffcceec7f93d5b1b587670f59915495d48e Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Wed, 27 Jan 2021 21:19:12 +0100
Subject: [PATCH 1/1] doc: return value exception command

If the exception cannot be raised, the command returns.
Currently the return values are not all the same.

Remove the sub-chapter 'Return value'

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 doc/usage/exception.rst | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/doc/usage/exception.rst b/doc/usage/exception.rst
index 412a03ba0f..db1490f005 100644
--- a/doc/usage/exception.rst
+++ b/doc/usage/exception.rst
@@ -61,8 +61,3 @@ Examples
     pc = 0x56076dd1a0f9, pc_reloc = 0x540f9
 
     resetting ...
-
-Return value
-------------
-
-The return value $? is always set to 0 (true).
-- 
2.29.2

